client/my-sites/site-settings/controller.js

```diff
@@ -1,26 +1,24 @@
 /**
  * External Dependencies
  */
-import ReactDom from 'react-dom';
-import React from 'react';
-import page from 'page';
 import i18n from 'i18n-calypso';
+import page from 'page';
+import React from 'react';
 
 /**
  * Internal Dependencies
  */
-import sitesFactory from 'lib/sites-list';
-import route from 'lib/route';
-import { renderWithReduxStore } from 'lib/react-helpers';
-import config from 'config';
 import analytics from 'lib/analytics';
-import titlecase from 'to-title-case';
-import SitePurchasesData from 'components/data/purchases/site-purchases';
-import { SiteSettingsComponent } from 'my-sites/site-settings/main';
+import config from 'config';
 import DeleteSite from './delete-site';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import route from 'lib/route';
+import SiteSettingsComponent from 'my-sites/site-settings/main';
```

Comment:
Why change from a destructured import? `SiteSettingsComponent` isn't exported as default as far as I can tell, but there may be a rule I don't know about because it is a class.